index.html

```diff
@@ -47,9 +47,17 @@ <h1 class="title has-text-black">Jamz</h1>
         </div>
       </div>
     </div>
-  </div>
-  <script defer src="https://use.fontawesome.com/releases/v5.14.0/js/all.js"></script>
-  <script src="./assets/script/script.js"></script>
+    <<<<<<< HEAD <script defer src="https://use.fontawesome.com/releases/v5.14.0/js/all.js">
+      </script>
+      <script src="./assets/script/script.js"></script>
 </body>
 
 </html>
+=======
+</div>
+<script defer src="https://use.fontawesome.com/releases/v5.14.0/js/all.js"></script>
+<script src="./assets/script/script.js"></script>
+</body>
+
+</html>
+>>>>>>> origin/development
```

Comment:
This was somehow left over from the merge conflict resolution, so need to remove the "<<<< HEAD", "=========="  and ">>>>>> origin development" text on lines 50, 56, and 63.